airbyte-integrations/connectors/source-facebook-marketing/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_facebook_marketing"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "==0.62.1"
+airbyte-cdk = "^0"
```

Comment:
This will continue to lock the updates up to the `1.0.0`, is it safe to set, when it comes to the `latest` CDK version?
I would rather put `>=0.62.1`, which will start from `0.62.0` as a regular stable working version and check the latest version as well. 

If there is a strong reason why would we use exactly `^0`, it's good to know. Thanks.

Reply:
I believe `>=0.x.x` will actually continue to get breaking changes from `1.0.0` and onward. Even if we aren't really adhering to true breaking change semantic versioning, it would be good to protect ourselves from that. `^0 and `~=0.69.1` would be behaviorally equivalent as far as I can tell